docker/docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+---
+version: "3"
+services:
+  storywise:
+    image: mihainueleanu/storywise:latest
+    container_name: storywise
+    hostname: storywise
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/Brussels
+      - MONGODB_URI=mongodb://${MONGO_INITDB_ROOT_USERNAME}:${MONGO_INITDB_ROOT_PASSWORD}@mongodb:27017/
+      - DATABASE_NAME=storywise
+      - USERNAME=${USERNAME}
+      - PASSWORD=${PASSWORD}
+      - API_BASE_URL=${API_BASE_URL}
+    networks:
+      - storywise
+    depends_on:
+      - mongodb
+    ports:
+      - 9280:3000
```

Comment:
could the port be parametrised via env variable, with a default to 3000?

Reply:
I can check that.. I will look at that asap. 
Also, fyi, I've adapted also the Dockerfile on my side to build a ARM compatible image. If you're interested. I can add it as well.